src/com/superiornetworks/icarus/commands/Command_smite.java

```diff
@@ -48,8 +52,8 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
         player.setHealth(0D);
         player.setOp(false);
         player.setGameMode(GameMode.SURVIVAL);
-        Bukkit.broadcastMessage(ChatColor.RED + player.getName() + " has been very naughty, indeed.");
-        player.sendMessage(ChatColor.RED + "You were smitten by: " + sender.getName());
+        Bukkit.broadcastMessage(ChatColor.RED + player.getName() + " has been very naughty, indeed.\nReason - " + ChatColor.YELLOW + smiteReason);
+        player.sendMessage(ChatColor.RED + "You were smitten by: " + sender.getName() ", Reason - " + ChatColor.YELLOW + smiteReason);
```

Comment:
As reasons will probably be optional the reason part should only show of a reason was given. Same for the line above.